rfcs/0053-reducing-daemon-memory-usage.md

```diff
@@ -0,0 +1,137 @@
+## Summary
+[summary]: #summary
+
+This RFC proposes changes to the Berkeley release of the daemon which will bring daemon's maximum possible memory usage within the range of our current hardware memory requirements of the Mainnet release.
+
+## Motivation
+[motivation]: #motivation
+
+With zkApps enabled, the maximum memory usage of the daemon now exceeds the current hardware requirements (`16GB`), in the event that the network is fully saturated with max-cost zkApps transactions in blocks for an extended period of time. With the planned parameters for Berkeley, we estimate that the maximum memory usage of a daemon is around `58.144633GB`.
+
+## Detailed design
```

Comment:
could you include more details about the relevant code changes? What will the lmdb abstraction look like? Can you share an example section of code to-be-changed?

Reply:
I am specifically curious how/if at all this integrates with the `Cached.t` infrastructure

Reply:
I'm adding a section which details the new disk caching interface and how it will be used.